warp10/src/main/java/io/warp10/continuum/ingress/DatalogForwarder.java

```diff
@@ -267,10 +267,18 @@ private boolean doUpdate(DatalogAction action) {
           if (action.request.getAttributesSize() > 0) {
             String accel = "";
             if (null != action.request.getAttributes().get(StandaloneAcceleratedStoreClient.ATTR_NOCACHE)) {
-              accel += StandaloneAcceleratedStoreClient.NOCACHE + " ";
+              if ("false".equals(action.request.getAttributes().get(StandaloneAcceleratedStoreClient.ATTR_NOCACHE))) {
+                accel += StandaloneAcceleratedStoreClient.CACHE + " ";                
+              } else {
+                accel += StandaloneAcceleratedStoreClient.NOCACHE + " ";
+              }
             }
             if (null != action.request.getAttributes().get(StandaloneAcceleratedStoreClient.ATTR_NOPERSIST)) {
-              accel += StandaloneAcceleratedStoreClient.NOPERSIST;              
+              if ("false".equals(action.request.getAttributes().get(StandaloneAcceleratedStoreClient.ATTR_NOPERSIST))) {
```

Comment:
In DELETE, FETCH and UPDATE, the test is `Boolean.TRUE.equals` so for coherency it should be `"true".equals` and switch if/else.